controllers/suite_test.go

```diff
@@ -236,9 +236,7 @@ func newLoadTest() *grpcv1.LoadTest {
 				BigQueryTable: &bigQueryTable,
 			},
 
-			Scenarios: []grpcv1.Scenario{
-				{Name: "cpp-example-scenario"},
-			},
+			ScenariosJSON: "{\"name\": \"cpp-example-scenario\"}",
```

Comment:
it feels odd that you're assigning something that actually doesn't have the prescribed format.
the top-level `{ "scenarios": [` seems missing.
Also below.

Reply:
@jtattermusch I agree 💯%. I want to improve the testing suite in general. I will likely follow this PR up with changes for the test suite and better tests, because it will be a substantially large change.